CHANGELOG.md

```diff
@@ -6,7 +6,7 @@ All notable changes to this project will be documented in this file.
 ### Added
 
 ### Fixed
-
+-   #1355 : remove BasicParser arguments.
```

Comment:
I think this should go in "Changed" not "Fixed" as nothing was broken

Reply:
ok